docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -1054,6 +1054,28 @@ When I save number of open tabs to $scopes variable `$variableName`
 When I save number of open tabs to SCENARIO variable `count`
 ----
 
+=== Alert Steps
+
+==== Type text in the alert input field
+
+This step allows you to type text into the alert input field and then either accept or dismiss the alert.
+
+An *alert* is a small dialog box that appears on the display screen to provide information or to warn about a potentially damaging operation. To proceed, the user needs to confirm or dismiss the alert by clicking the 'OK' or 'Cancel' button, respectively.
```

Comment:
add note that this step will not work with authorization alerts, use URL auth insteadd